src/index.js

```diff
@@ -1,28 +1,134 @@
 /* eslint-disable */
+
 import './style.css';
+import interactive from './interactiveList.js';
+import clearCompletedTasks from './addRemove.js';
+
+class SingleToDo {
+    constructor(description) {
+        this.description = description;
+        this.completed = false;
+        this.index = 0;
+    }
+}
+class ToDoList {
+    constructor(toDoTasksArray = [], container) {
+        this.toDoTasksArray = toDoTasksArray;
+        this.container = document.querySelector(container);
+    }
```

Comment:
- [ ] You have implemented methods to add and remove to-do items 👏 Well done. Now, please implement a method for editing a todo item because a user currently cannot edit a to-do item 🙏